packages/jscpd/__tests__/formats.test.ts

```diff
@@ -75,12 +75,12 @@ describe('jscpd formats', () => {
 		'Scripts': [
 			{
 				name: 'javascript',
-				clonesCount: 5,
+				clonesCount: 9,
 				folder: pathToFixtures + '/javascript',
 			},
 			{
 				name: 'typescript',
-				clonesCount: 2,
+				clonesCount: 6,
 				folder: pathToFixtures + '/javascript',
```

Comment:
I'm not sure what's going on here, but it looks like the tests generally failed in https://github.com/jsoref/jscpd/actions/runs/3842735915, w/o me really changing the code itself...